lib/cli/index.js

```diff
@@ -32,6 +32,7 @@ exports.run = () => {
         .allowUnknownOption(true)
         .option('-r, --reporter <reporter>', 'test result reporter (flat by default)', collect)
         .option('--html-reporter-path <path>', 'relative path where html reporters will be stored', collect)
+        .option('--html-failed-only', 'html report generates only for failed tests')
```

Comment:
If this is desired behavior, I can update the code.

Btw. It is greate to have it configurable, but I don't think that any other option would anyone want to use, since i cannot see any real scenario, where i would want to generate screens of passed/skipped tests. I usually don't care about these tests.